source/playbooks/metrics-and-goals.rst

```diff
@@ -1,5 +1,5 @@
-Retrospectives
-==============
+Metrics and goals
```

Comment:
Change "Configuring retrospectives before a run" to "Configure retrospectives before a run" (remove gerund)

Reply:
Consider revisiting the headings on this page. It may not be necessary to go down to H4